### DISCUSSION TITLE
An Option to replace the hyphen ("-") with an m-dash (—)

### DISCUSSION THREAD

**Discussion by akhilesh-balaji** [monkeytypegame/monkeytype#1532] [Ideas] [answered]
When the punctuation mode is enabled, it would be nice to have a settings option to replace the character sequence " - “, as in “Please call my lawyer - Richard Smith - on Tuesday” with an em dash (“Please call my lawyer—Richard Smith—on Tuesday”). I use the m-dash more often (I've set up a custom hotkey for it), and it would be nice to have this option in the settings.

**Comment by Otard95** [comment, marked as answer]
If I remember correctly, you should be able to type an m-dash for any normal dash, as the input logic will treat your typed m-dash as a normal one. So say your test is "a-z" you should be able to type "a—z" and complete the test with 100% accuracy.

**Reply by Otard95** [reply]
@akhilesh-balaji If this was a satisfactory. You can mark the comment as the answer so that other can see at a glance that this discussion is answered.

**Reply by akhilesh-balaji** [reply]
@Otard95 done, thanks!
